README.md

```diff
@@ -4,87 +4,74 @@
-
-After we create a Session we can send any of the following `Get, Post, Put, Patch, Options, Delete, and Head` requests. In the above example we send a 
-GET request to the session with the alias _google_ and check the HTTP response code. Then send a another GET request but this time to the session with 
-the alias _github_ and pass in a `uri`. In this case it is `/users/bulkan` which will return a JSON string. `RequestsLibrary` returned object provides 
-a method to get the content as a JSON object format called json().
+After we create a Session we can send any of the following `Get, Post, Put, Patch, Options, Delete, and Head` requests. In the above example we send a GET request to the session with the alias _google_ and check the HTTP response code. Then send a another GET request but this time to the session with the alias _github_ and pass in a `uri`. In this case it is `/users/bulkan` which will return a JSON string. `RequestsLibrary` returned object provides a method to get the content as a JSON object format called json().
 
 Here is another test case where an outbound http proxy is used.
 
-|                           |                                  |                     |                                |                      |
-| :------------------------ | :------------------------------- | :--------------     | :----------------------------- | :------------------- |
-| Settings                  |                                  |                     |                                |                      | 
-| Library                   | RequestsLibrary                  |                     |                                |                      |
-| Test Cases                |                                  |                     |                                |                      |
-| Proxy Requests            |                                  |                     |                                |
-|                           | ${proxies}=                      | Create Dictionary   | http=http://acme.com:912       | https=http://acme.com:913 |
-|                           | Create Session                   | github              | http://api.github.com          | proxies=${proxies}    |
-|                           | ${resp}=                         | Get Request         | google                         | /                    |
-|                           | Should Be Equal As Strings       | ${resp.status_code} | 200                            |                      |
-
+|                  |                            |                     |                          |                           |
+| ---------------- | -------------------------- | ------------------- | ------------------------ | ------------------------- |
+| *** Settings *** |                            |                     |                          |                           |
```

Comment:
@nizish - my edits didn't change content per se, and this is from the original. However, I think you are correct - it should say `github`.